once/Dockerfile

```diff
@@ -151,7 +151,7 @@ COPY ./custombuild/ ./custombuild/
 # Let //go:embed to embed the built static files
 COPY --from=authgear-portal-stage-0 /usr/src/app/resources/authgear/ ./resources/authgear/
 COPY --from=authgear-portal-stage-1 /usr/src/app/dist/ ./resources/portal/static/
-RUN make build BIN_NAME=authgear-portal TARGET=portal GIT_HASH=$GIT_HASH
+RUN make build AUTHGEARONCE=1 BIN_NAME=authgear-portal TARGET=portal GIT_HASH=$GIT_HASH
```

Comment:
I think this should be done by ARG
https://docs.docker.com/reference/dockerfile/#arg

Reply:
Since this is the Dockerfile to build once, AUTHGEARONCE should always be set. If it is a ARG, then it depends on the caller to set it correctly, which is no good.

Reply:
I have this comment because I saw this file is generated from the authgear & portal dockerfile, with some string replacement which looks like easy to break. And modifier of authgear Dockerfile & portal Dockerfile might not know this command is related to the once Dockerfile.

Another suggestion:
Use `ENV AUTHGEARONCE=1` on top of the once Dockerfile, so that the builds automatically have `AUTHGEARONCE=1` set and we don't have to do string replacements.

Reply:
Added two commits to address this.

I review the official doc on Dockerfile again, and it is better to use `ARG` with default value.